alfalfa_client/alfalfa_client.py

```diff
@@ -12,7 +13,8 @@ class AlfalfaClient:
     # default should be http://localhost/api
     def __init__(self, url='http://localhost'):
         self.url = url
-        self.haystack_filter = self.url + '/api/read?filter='
+        self.api_read_filter = self.url + '/api/read?filter='
```

Comment:
I've been moving to f-strings as much as possible... so f'{self.url}/api/read?filter='.

But that can be done slowly.